src/data/code-samples/en/accessibility-language/swift-ui.md

```diff
@@ -1,14 +1,19 @@
 # Accessibility language - SwiftUI
 
-On iOS, the [`accessibilitySpeechLanguage`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/1620188-accessibilityspeechlanguage) key of [`AttributedString`](https://developer.apple.com/documentation/foundation/attributedstring) can be used to speak content in a specific language. Multiple `AttributedString`s can be embedded inside a single `AttributedString` by using the `+` operator to speak content in multiple languages.
+On SwiftUI, the [`accessibilitySpeechLanguage`](https://developer.apple.com/documentation/foundation/nsattributedstring/key/1620188-accessibilityspeechlanguage) key of [`AttributedString`](https://developer.apple.com/documentation/foundation/attributedstring) can be used to speak content in a specific language. Multiple `AttributedString`s can be embedded inside a single `AttributedString` by using the `+` operator to speak content in multiple languages.
 
 ```swift
-let attributedString = AttributedString("Appt", attributes: AttributeContainer([.accessibilitySpeechLanguage: "nl_NL"]))
+let attributedString = AttributedString(
+    "Appt",
+    attributes: AttributeContainer([
+        .accessibilitySpeechLanguage: "nl_NL"
+    ])
+)
 
 Text(attributedString)
 ```
 
-In SwiftUI, you can easily add localized accessibility features to any view using the [accessibilityLabel](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv) and [accessibilityHint](https://developer.apple.com/documentation/swiftui/view/accessibilityhint(_:)-3i2vu) modifiers. These modifiers accept a [LocalizedStringKey](https://developer.apple.com/documentation/swiftui/localizedstringkey), making it straightforward to provide localized descriptions and hints for your UI components. This simplifies the process of supporting multiple languages in your app's UI and accessibility technologies.
+In SwiftUI, you can also easily add localized accessibility features to any view using the [accessibilityLabel](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv) and [accessibilityHint](https://developer.apple.com/documentation/swiftui/view/accessibilityhint(_:)-3i2vu) modifiers. These modifiers accept a [LocalizedStringKey](https://developer.apple.com/documentation/swiftui/localizedstringkey), making it straightforward to provide localized descriptions and hints for your UI components. This simplifies the process of supporting multiple languages in your app's UI and accessibility technologies.
```

Comment:
Hi Yurii, can you add ticks in the links to all SwiftUI developer properties, functions, etc. If the links go to an developer docs article, you don't need to add the ticks. Only for the pure "developer" links.

```
 [accessibilityLabel](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv)
```

change to

```
 [`accessibilityLabel`](https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv)
```

Can you also verify this for the other code samples. Most seem to have the ticks in place already.

Reply:
@JJdeGroot went through SwiftUI code samples and corrected ticks for developer links